deploy.yml

```diff
@@ -0,0 +1,225 @@
+    params:
+      include_source_tarball: true
+    trigger: true
+  - task: Build Image
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: onsdigital/eq-python-build
+      inputs:
+      - name: eq-survey-runner-pre-release
+      outputs:
+      - name: compiled-eq-survey-runner
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          cd eq-survey-runner-pre-release
+
+          mkdir eq-survey-runner
+          tar -xzf source.tar.gz -C eq-survey-runner --strip-components=1
+
+          cd eq-survey-runner
+          pipenv install --dev
+          yarn compile
+
+          cp -R ../eq-survey-runner/* ../../compiled-eq-survey-runner
+          cp ../tag ../../compiled-eq-survey-runner/.application-version
+
+
+  - put: survey-runner-image
+    params:
+      build: compiled-eq-survey-runner
+      tag: eq-survey-runner-pre-release/tag
+    get_params:
+      skip_download: true
+  - put: survey-runner-static-image
+    params:
+      build: compiled-eq-survey-runner
+      dockerfile: compiled-eq-survey-runner/Dockerfile.static
+      tag: eq-survey-runner-pre-release/tag
+    get_params:
+      skip_download: true
+
+- name: pre-prod-deploy
+  max_in_flight: 1
+  plan:
+  - get: eq-survey-runner-pre-release
+    passed: [build-eq-survey-runner]
+    params:
+      include_source_tarball: true
+    trigger: true
+    params:
+      include_source_tarball: true
+  - get: eq-ecs-deploy
+  - task: Deploy Survey Runner
+    params:
+      AWS_ACCESS_KEY_ID: ((preprod_aws_access_key))
+      AWS_SECRET_ACCESS_KEY: ((preprod_aws_secret_key))
+      TF_VAR_env: 'preprod-new'
+      TF_VAR_aws_access_key: '((preprod_aws_access_key))'
+      TF_VAR_aws_secret_key: '((preprod_aws_secret_key))'
+      TF_VAR_vpc_id: '((preprod_vpc_id))'
+      TF_VAR_ecs_cluster_name: 'preprod-eq'
+      TF_VAR_docker_registry: {{docker_registry}}
+      TF_VAR_aws_alb_arn: {{preprod_aws_alb_arn}}
+      TF_VAR_aws_alb_listener_arn: {{preprod_aws_alb_listener_arn}}
+      TF_VAR_service_name: 'surveys'
+      TF_VAR_container_name: 'eq-survey-runner'
+      TF_VAR_container_port: 5000
+      TF_VAR_listener_rule_priority: 10
+      TF_VAR_task_has_iam_policy: true
+      TF_VAR_slack_alert_sns_arn: '((preprod_slack_alert_sns_arn))'
+      TF_VAR_healthcheck_path: '/status'
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: onsdigital/eq-terraform-build
+      inputs:
+      - name: eq-survey-runner-pre-release
+      - name: eq-ecs-deploy
+      run:
+        path: bash
+        args:
+        - -exc
+        - |
+          cd eq-ecs-deploy
+          tfenv install
+
+          release_version=$(cat ../eq-survey-runner-pre-release/tag)
+
+          terraform init -backend-config="bucket="concourse-preprod-terraform-state"" -backend-config="key="preprod-ecs-runner""
```

Comment:
@jonnyshaw89 Do we not need to do a `plan`?

Reply:
Plan is pointless as this is automated, so what will we do with the output?

Reply:
Yeah, good point. I kinda thought that too. Just wanted to double check